src/translations/wishlist.translations.js

```diff
@@ -0,0 +1,32 @@
+export const WISHLIST_TITTLES = {
+  0: {
+    empty: 'Ваш список уподобань порожній',
+    filled: 'Список уподобань'
+  },
+  1: {
+    empty: 'Your wishlist is empty',
+    filled: 'Wishlist'
+  }
+};
+
+export const WISHLIST_BUTTONS = {
+  0: {
+    empty: 'ОБРАТИ ЗАРАЗ',
```

Comment:
Please, lowercase translations, the casing should be defined inside styles for a particular button. Here we define actual data and we shouldn't care how it is going to be represented